### DISCUSSION TITLE
No images from NOAA

### DISCUSSION THREAD

**Discussion by ON6PH** [jekhokie/raspberry-noaa-v2#155] [Q&A]
For a couple of days now i am not receiving any NOAA images. Meteor works fine, so there is no problem with antenna or receiver. Could an update to the software have any influence here? In my .noaa.conf i have set the receiver for Meteor to RTL_SDR. Not sure if NOAA uses this also or is he using gnuradio? How can i check...

![NOAA-15-20210219-081026-MCIR](https://user-images.githubusercontent.com/35851731/108516795-eff78800-72c6-11eb-8780-93b8ab5892d6.jpg)

**Comment by dom-robinson** [comment]
This might help:

1. scheduler.sh sets up the ‘at’ queue to call either receive_meteor.sh or receive_noaa.sh on schedule.
2. receive_noaa.sh currently always uses rtf_fm and captures a .wav, converts using wxtoimg to the nova images.
3. receive_meteor.sh has a switch for gnuradio or rtf_rm (read from your settings) that then EITHER
	- uses rtf_fm to capture a .wav and follows similar processing to the noaa
	- OR uses the gnuradio filter, expressed in python, called rtlsdr_m2_lrpt_rx.py

So its really unlikely the gnuradio setting will have any effect on NOAA captures..

-- 
Dom Robinson

**Reply by ON6PH** [reply]
So, in my case rtl_fm is used for both NOAA and Meteor and still i receive nothing on NOAA and good images on Meteor... I think i'll create a new image for he Pi. The setup is easy with this V2 ;-)

**Comment by RajSinghBains** [comment]
hi guys, did you ever figure out what was causing this. i seem to also have the same issue with my NOAA images, and ime not too sure what is causing it or how to fix it.

regards, Raj.
